images/prometheus-config-reloader/tests/main.tf

```diff
@@ -38,9 +38,17 @@ resource "helm_release" "kube-prometheus-stack" {
   }
 }
 
-module "helm_cleanup" {
-  source    = "../../../tflib/helm-cleanup"
-  name      = helm_release.kube-prometheus-stack.id
-  namespace = helm_release.kube-prometheus-stack.namespace
+data "oci_exec_test" "helm_cleanup" {
```

Comment:
Worth noting this still requires the test to be serialized, which means using the `flock` trick we use with tigera-operator's tests.

It might just make sense to only test one variant, and skip it for others, or use the `flock` trick and serialize them and clean them up fully after each run.

@joshrwolf has ideas in the works about how to make this all _much_ better and more hermetic, but until then we only have growing piles of band-aids :-/

Reply:
ACK, thought we just needed the cleanup part but makes sense, was hoping to not have to move the helm test into a shell script, but if thats the only option then I guess we can. Might ask @mritunjaysharma394 to close out this part tomorrow